### ISSUE TITLE
The SSE _mm_invsqrt_ps intrinsic is missing

### ISSUE THREAD

**Issue by gnzlbg** [rust-lang/stdarch#438]
See: https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_invsqrt_ps&expand=4513,3028

@alexcrichton is it possible to generate a list of the missing intrinsics from stdsimd-verify ? I wonder if we haven't missed other intrinsics.

---

EDIT: _mm_invsqrt_pd appears to be missing as well.

**Comment by alexcrichton** [comment]
Some of the missing ones are ones that gcc/clang don't implement becuase there's like some support library that Intel's compiler only supports, I'm not sure if this falls in that category

**Comment by gnzlbg** [comment]
Yeah, clang doesn't implement these either, but that's a bug in clang.

**Comment by sayantn** [comment]
These are SVML intrinsics, which is Intel C compiler specific, so we can't use them
